haddock-api/haddock-api.cabal

```diff
@@ -89,6 +96,37 @@ library
     Haddock.Convert
     Paths_haddock_api
 
+test-suite spec
+  type:             exitcode-stdio-1.0
+  default-language: Haskell2010
+  main-is:          Spec.hs
+  ghc-options: -Wall
+
+  hs-source-dirs:
+      test
+    , src
+
+  other-modules:
+    Haddock.Backends.Hyperlinker.ParserSpec
+
+  build-depends:
+      base >= 4.3 && < 4.9
+    , bytestring
+    , filepath
+    , directory
+    , containers
+    , deepseq
+    , array
+    , xhtml >= 3000.2 && < 3000.3
+    , Cabal >= 1.10
+    , ghc >= 7.10 && < 7.10.2
```

Comment:
I think relaxing this upper bound would be good, perhaps to something like `7.10.*`. Unless we have a reason to believe that it won't work with 7.10.2, there's no need to constrain it this tightly. In all likelyhood, this will come out after 7.10.2 is released anyway.